livre/eq/zeq/communicator.cpp

```diff
@@ -259,7 +259,7 @@ class Communicator::Impl
         // Receiving all queued events from all receivers without blocking.
         for( auto subscriber : subscribers )
             while( subscriber->receive( 0 ))
-                /*nop*/ ;
+                _config.sendEvent( REDRAW );
```

Comment:
I am seeing those kind of updates may be dangerous for animation mode. Has to be carefully tested.